qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -952,6 +939,13 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                                                 sigout-1, 1, 'range'),
                                 vals=vals.Enum(0.075, 0.15, 0.75, 1.5))
 
+            self.add_parameter('signal_output{}_ampdef'.format(sigout),
+                               get_cmd=None, set_cmd=None,
+                               initial_value='Vpk',
+                               label="Signal output amplitude's definition",
+                               unit='V',
```

Comment:
this parameter does not have a reasonable unit.
```suggestion
                               unit='',
```